example/src/services/metamask.ts

```diff
@@ -4,6 +4,7 @@ declare global {
             isMetaMask: boolean;
             send: (params: any)=> Promise<any>;
             on: (eventName: any, callback: any) => any;
+            requestIndex: () => Promise<any>;
```

Comment:
Isn't this suppose to return PolkadotApi interface?

Reply:
No, here is an example from the official MetaMask documentation:

```js
async function requestIndex () {
  index = await ethereum.requestIndex()
  pluginApi = await index.getPluginApi(pluginOrigin)

  // Use the API provided by the Snap here!
  const pong = await pluginApi.ping()
}
```

Reply:
ok than it should be:
```javascript
requestIndex: () => Promise<{getPluginApi: () => Promise<PolkadotApi>}>
```